src/system-health/health_checker/manager.py

```diff
@@ -0,0 +1,82 @@
+class HealthCheckerManager(object):
+    STATE_BOOTING = 'booting'
+    STATE_RUNNING = 'running'
+    boot_timeout = None
+
+    def __init__(self):
+        self._checkers = []
+        self._state = self.STATE_BOOTING
+
+        from .config import Config
+        self.config = Config()
+        self.initialize()
+
+    def initialize(self):
+        from .service_checker import ServiceChecker
+        from .hardware_checker import HardwareChecker
+        self._checkers.append(ServiceChecker())
+        self._checkers.append(HardwareChecker())
+
+    def check(self, chassis):
+        from .health_checker import HealthChecker
+        HealthChecker.summary = HealthChecker.STATUS_OK
+        stats = {}
+        self.config.load_config()

```

Comment:
If the platform config file does not exist, what is the expected action next?

Reply:
We have default value for all configuration entries. If config file does not exist, default value will be used.